src/app/AnswerVerification.js

```diff
@@ -0,0 +1,3 @@
+function isAsnwerCorrect (correctAnswer, playerAnswer) {
+    return correctAnswer === playerAnswer;
```

Comment:
@KlaraGajaszek @maciekkwiet jakim typem danych będzie odpowiedź? Na ten moment założyłam, że stringiem

Reply:
Funkcja zwraca true/false, a więc nie ma to w tym przypadku raczej znaczenia.
Ważne jest żeby oba argumenty były tego samego typu.